packages/console/src/pages/SignInExperience/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx

```diff
@@ -196,7 +196,7 @@ const LanguageDetails = () => {
                         }
                       }}
                     >
-                      <Clear />
+                      <Clear className={style.clearIcon} />
```

Comment:
nit: Well, you don't have to add this new class name, but instead apply the styles by using its parent class name. e.g.
```
.clearButton {
  ...

  svg {
    width: 16px;
    height: 16px;
  }
}
```